pkg/proxy/ipvs/proxier.go

```diff
@@ -1131,14 +1131,18 @@ func (proxier *Proxier) syncProxyRules() {
 		} else {
 			nodeAddresses = nodeAddrSet.List()
 			for _, address := range nodeAddresses {
+				a := net.ParseIP(address)
+				if a.IsLoopback() {
+					continue
+				}
```

Comment:
Should we reject the loopbacks in validation too? or is excessive? `proxier.nodePortAddresses`
https://github.com/kubernetes/kubernetes/blob/57459ebd3e56eb0747e4697422e8ac24f7d113c6/pkg/proxy/apis/config/validation/validation.go#L107

Reply:
No, it is a valid use-case now (that we are using), and it is still necessary for proxy-mode=iptables. The reason for adding loopback in NodePortAddresses is to limit/disable nodeports, not that anybody _wants_ to use loopback for nodeport. I think have different validations for different proxy-modes must be avoided.